test/phpunit/PdfDocTest.php

```diff
@@ -141,13 +141,14 @@ public function testPdfDocGetLineDesc()
 		$db=$this->savdb;
 
 		$localproduct=new Product($this->savdb);
-		$result = $localproduct->fetch(0, 'PIDRESS');
+		$localproduct->initAsSpecimen();
```

Comment:
Why a initAsSpecimen() to set dummy properties if just after there is a fetch to load real data from database ?

Reply:
Sorry I tried something but forgot to delete it